### PR TITLE
Turn samples into challenge templates

### PR DESCRIPTION
**Pull request by sroettger**
I removed:
* bash
* apache-php

Renamed:
* apache-others to web
* challenge-skeleton to pwn (and made it a chal.c)

You can choose the template using `kctf chal create --template foo` (default is pwn since pwn is best challenge ;) )

### CODE REVIEW

File: dist/bin/kctf-challenge [hunk 1]
```diff
@@ -370,13 +370,14 @@ function kctf_chal_create_usage {
   echo "usage: kctf chal create [args] name" >&2
   echo "args:" >&2
   echo "  -h|--help       print this help" >&2
+  echo "  --template      which template to use (run --template list to print available templates)" >&2
```
Comment (sirdarckcat) [inline dist/bin/kctf-challenge:366]: mention pwn is default?

File: dist/challenge-templates/pwn/challenge.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: kctf.dev/v1alpha1
 kind: Challenge
 metadata:
-  name: mychal
+  name: pwn
```
Comment (sirdarckcat) [inline dist/challenge-templates/pwn/challenge.yaml:4]: should the name be set with `kctf chal create`? I didnt find where this happens

**Review by sirdarckcat — APPROVED** [review]

File: dist/challenge-templates/pwn/challenge.yaml (hunk 2 above)
Comment (sirdarckcat) [reply dist/challenge-templates/pwn/challenge.yaml:4]: found it

https://github.com/google/kctf/blob/d35b42f705db3d481a93831be2c03a4117d6c334/dist/bin/kctf-challenge#L461

**Comment by sirdarckcat** [comment]
ah this isnt gonna work

https://github.com/google/kctf/blob/beta/.github/workflows/update-images.yaml#L161

**Comment by sroettger** [comment]
> ah this isnt gonna work
> 
> https://github.com/google/kctf/blob/beta/.github/workflows/update-images.yaml#L161

What do you mean?

**Comment by sirdarckcat** [comment]
oh, I missed you moved them to /tmp

**Pull request merged**